### PR TITLE
Added an option to theme the CircularProgressBar success icon

### PR DESCRIPTION
**Pull request by dimafeldman**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6251

File: src/ThemeProvider/test/ThemeProvider.tsx [hunk 1]
```diff
@@ -21,6 +23,11 @@ function themeProviderWithAllProps() {
             medium: Close,
             large: Close,
           },
+          CircleLoaderCheck: {
```
Comment (shlomitc) [inline src/ThemeProvider/test/ThemeProvider.tsx:26]: this should be named the same as the component name and not as the icon name

File: src/Themes/businessDashboard/icons/CircleLoaderCheck.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    height="14px"
+    viewBox="0 0 20 14"
+    version="1.1"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <g stroke="none" strokeWidth="1" fill="none" fillRule="evenodd">
+      <g transform="translate(-18.000000, -21.000000)" fill="currentcolor">
+        <path
+          d="M36.3021353,21.2837704 C36.697698,20.8983504 37.3308095,20.9065726 37.7162296,21.3021353 C38.0720019,21.6672701 38.0923631,22.2348134 37.7822697,22.6231105 L37.6978647,22.7162296 L25.3820753,34.7162296 C25.0236284,35.0654855 24.468543,35.0923513 24.0798441,34.7968271 L23.9863458,34.7162296 L18.3021353,29.177768 C17.9065726,28.792348 17.8983504,28.1592364 18.2837704,27.7636737 C18.6395428,27.398539 19.2063659,27.3634457 19.6025851,27.6633507 L19.6978647,27.7453089 L24.684,32.603 L36.3021353,21.2837704 Z"
+          id="Path-2"
```
Comment (shlomitc) [inline src/Themes/businessDashboard/icons/CircleLoaderCheck.js:16]: remove

File: src/Themes/businessDashboard/icons/CircleLoaderCheck.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  <svg
+    className={className}
+    width="20px"
+    height="14px"
+    viewBox="0 0 20 14"
+    version="1.1"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <g stroke="none" strokeWidth="1" fill="none" fillRule="evenodd">
+      <g transform="translate(-18.000000, -21.000000)" fill="currentcolor">
```
Comment (shlomitc) [inline src/Themes/businessDashboard/icons/CircleLoaderCheck.js:13]: remove all properties. maybe the "transform" can stay but, but all fill and stroke rules should be removed.
The guidelines for a well-structured SVG can be found here: https://github.com/wix/wix-ui/tree/master/packages/wix-ui-icons-common#guidelines

File: src/Themes/businessDashboard/icons/CircleLoaderCheck.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+export default ({ className }) => (
+  <svg
```
Comment (shlomitc) [inline src/Themes/businessDashboard/icons/CircleLoaderCheck.js:4]: add `fill="currentcolor"`

**Review by shlomitc — CHANGES_REQUESTED** [review]

File: src/ThemeProvider/test/ThemeProvider.tsx (hunk 1 above)
Comment (dimafeldman) [reply src/ThemeProvider/test/ThemeProvider.tsx]: updated. but what about cases where we will need to update multiple icons for the same component? a multiple level object?

Comment (shlomitc) [reply src/ThemeProvider/test/ThemeProvider.tsx]: in that case, we nest the different icons. for example: https://github.com/wix/wix-style-react/blob/95bf74694cb0b9c3d44d5f2dd118405c4a27939a/src/ThemeProvider/index.d.ts#L12

**Pull request merged**